### PR TITLE
update guild leaders

### PR DESCRIPTION
**Pull request by eric-w-hart**

### CODE REVIEW

**Comment by eric-w-hart** [comment]
@bmccord or @woodrbe can you review and approve?

**Review by amyschoen — APPROVED** [review]
👍

File: optumdocs/docs/Guilds.md [hunk 1]
```diff
@@ -11,5 +11,5 @@ sidebar_label: Guilds
 | Compute            |                                           Softwared across the full stack spectrum                                            | Ben Woodring             |
 | Data/ML/AI         |         Wrangling transformation and custom ML/AI solutions pertinent for managed data well and healthcare innovation         | Kevin Muse, Ben Woodring |
 | Sourcing           |       Best Practices and process improvements in service of code and development. Developer and open source experience        | Devin Houde, Amy Schoen  |
-| Clinical Solutions | An open source industry differentiator. Exploring/building technological solutions that advances the global healthcare agenda | Kevin Muse               |
-| Security           |                    Security solutions and security guidance for how we open source across the organization                    | Eric Hart, Devin Houde   |
+| Clinical Solutions | An open source industry differentiator. Exploring/building technological solutions that advances the global healthcare agenda | Keven Muse               |
```
Comment (amyschoen) [inline optumdocs/docs/Guilds.md:14]: @eric-w-hart do we have a new lead designated for Clinical?  We might as well update that as a part of this change as well.

**Comment by eric-w-hart** [comment]
@amyschoen I have updated the leaders. Rob Crockett will now take over for Clinical.

**Pull request merged**